pymc_marketing/clv/models/beta_geo.py

```diff
@@ -92,44 +101,35 @@ class BetaGeoModel(CLVModel):
                 "cores": 2,
             },
         )
-        model.build_model()
+
+        # The default 'mcmc' fit_method provides informative predictions and reliable performance on small datasets
         model.fit()
         print(model.fit_summary())
 
-    Predicting the customer-specific number of purchases for a future time
-    interval of length t given their previous frequency, recency, and T
-
-    .. code-block:: python
-
-        future_t = [2, 5, 7, 10]
-
-        expected_num_purchases = model.expected_num_purchases(
-            t=future_t,
-            customer_id=data["customer_id"],
-            frequency=data["frequenc"y],
-            recency=data["recency"],
-            T=data["T"],
-        )
+        # Maximum a Posteriori can quickly fit a model to large datasets,
+        # but will give limited insights into predictive uncertainty.
+        model.fit(fit_method='map')
+        print(model.fit_summary())
 
-    Estimating the expected number of purchases for a randomly chosen
-    individual in a future time period of length t
+        # Predict number of purchases for current customers over the next 10 time periods
+        expected_purchases = model.expected_purchases(future_t=10)
 
-    .. code-block:: python
+        # Predict probability customers are still active
+        probability_alive = model.expected_probability_alive()
 
-        expected_num_purchases_new_customer = model.expected_num_purchases_new_customer(
-            t=future_t,
-        )
+        # Predict number of purchases for a new customer over 't' time periods
+        expected_purchases_new_customer = model.expected_purchases_new_customer(t=10)
 
     References
     ----------
-    .. [1] Fader, P. S., Hardie, B. G., & Lee, K. L. (2005). “Counting your customers”
-           the easy way: An alternative to the Pareto/NBD model. Marketing science,
+    .. [1] Fader, P. S., Hardie, B. G., & Lee, K. L. (2005). “Counting your customers
```

Comment:
```suggestion
    .. [1] Fader, P. S., Hardie, B. G., & Lee, K. L. (2005). "Counting your customers
```
Seems like different kind of quote